storage/replica.go

```diff
@@ -202,7 +203,7 @@ type Replica struct {
 	// Last index applied to the state machine. Updated atomically.
 	appliedIndex uint64
 	configHashes map[int][]byte // Config map sha256 hashes @ last gossip
-	systemDBHash []byte         // sha256 hash of the system config @ last gossip
+	systemDBHash []byte         // sha1 hash of the system config @ last gossip
```

Comment:
precisely. we're doing checksums for de-duplication, so why use sha256? it's at least twice slower than sha1.
As for the other hash, it'll be going away with the ZoneConfig.